### ISSUE TITLE
Find is failing on a debian based container that has find

### ISSUE THREAD

**Issue by tas50** [mondoohq/cnquery#4894]
**Describe the bug**

Find is failing on the `nginx` container which is Debian based and has find.

```
docker run -it nginx /bin/bash
root@2c952940aee8:/# find . -name 'issue'
./etc/issue
```

![Image](https://github.com/user-attachments/assets/7688d60a-699e-41ec-ae52-a7a8db5b0577)

To repro this you can enable the container secrets policy and then run `cnspec scan docker image nginx`

**Comment by AdamVB** [comment]
This seems to be failing for alpine as well

**Comment by imilchev** [comment]
Scanning a docker image is different from scanning a running container. The fact that a container image has the `find` binary means nothing when scanning just the container image. The bug was elsewhere. I fixed it in #4974
